### ISSUE TITLE
Error while running yarn dev

### ISSUE THREAD

**Issue by rustyy** [payloadcms/payload#968]
# Bug Report

While running `yarn dev` the following error occurs:

```
ERROR in ./node_modules/payload/dist/admin/components/elements/ReactSelect/index.js 3:0-89
Module not found: Error: Can't resolve 'react-sortable-hoc' in '/.../node_modules/payload/dist/admin/components/elements/ReactSelect'
```

## Expected Behavior

No error should be thrown.

## Current Behavior

Error is thrown.

## Possible Solution

Seems like react-sortable-hoc is set as dev-dependency, hence not available in the production build.

## Steps to Reproduce

1. run `npx create-payload-app`
2. run `yarn dev`

## Detailed Description

Seen in version 1.0.25

**Comment by rustyy** [comment]
Created PR #969

**Comment by rustyy** [comment]
fixed in v1.0.27
